### ISSUE TITLE
YOLOv7 minimum in training loss

### ISSUE THREAD

**Issue by cschnu** [AlexeyAB/darknet#8896]
Dear all,

training yolov4 with my own dataset shows final mAP values around 75% and an almost monotonic loss decay. I am very happy with the results.
However, I wanted to check if I can get even better results with yolov7. Using the same dataset, I see a minimum in the training loss at around 600 iterations, and then the loss settles at a higher value. Nevertheless, the mAP values are very good and approach 77%.

![yolov4 vs. yolov7](https://github.com/AlexeyAB/darknet/assets/166595450/0e160520-6405-4f64-ac78-61c08e1e3485)

Some additional information:
- My dataset has one class only. I followed the README to adapt the config files `yolov4-custom.cfg` and `yolov7.cfg` accordingly
- I used the pretrained weights `yolov4.conv.137` and `yolov7.conv.132`. The minimum in the loss curve is still there if I start training from scratch.
- I used the following commands for training:
```
./darknet detector train data/obj.data yolov4-custom.cfg yolov4.conv.137 -dont_show -map -mAP_epochs 1
./darknet detector train data/obj.data yolov7.cfg yolov7.conv.132 -dont_show -map -mAP_epochs 1
```
- I have played around with learning rate, decay, momentum, and burn-in, but this only shifts the minimum horizontally and/or makes it wider.

Does anyone have a clue why the training loss curves of yolov4 and yolov7 look so differently? The minimum in the loss curve really bothers me.

Thanks for your help!
